sdk/notificationhubs/notification-hubs/package.json

```diff
@@ -46,12 +46,11 @@
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build:samples": "echo Obsolete",
-    "build:test:browser": "tsc -p . && rollup -c rollup.test.config.js 2>&1",
+    "build:test:browser": "tsc -p . && rollup --bundleConfigAsCjs -c rollup.test.config.cjs 2>&1",
     "build:test:node": "tsc -p . && dev-tool run bundle",
-    "build:test": "tsc -p . && rollup -c rollup.test.config.js 2>&1",
+    "build:test": "tsc -p . && rollup --bundleConfigAsCjs -c rollup.test.config.cjs 2>&1",
     "build:types": "downlevel-dts types/latest/ types/3.1/",
-    "build:output": "node scripts/renameOutput.mjs",
-    "build": "npm run build:test && api-extractor run --local && npm run build:types && npm run build:output",
+    "build": "npm run build:test && api-extractor run --local && npm run build:types",
```

Comment:
we're starting to really feel the pain of ESM incompatibility here... I'll look into making our dev-tool bundled configs ESM compatible, because currently we are getting some weirdness with named exports.

Reply:
This is only for our rollup for testing purposes